File: SeatMe-Backend/src/main/java/edu/icet/ecom/dto/Table.java
package edu.icet.ecom.dto;

import lombok.*;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class Table {
    private Long id;
    private String tableNumber;
    private int capacity;
    private boolean available;
    private Long restaurantId; // optional, if linked to restaurant
}

File: SeatMe-Frontend/src/app/model/Table.ts
export class Table {
  id?: number;
  tableNumber: string;
  capacity: number;
  available: boolean;
  restaurantId: number;

  constructor(id: number, tableNumber: string, capacity: number, available: boolean, restaurantId: number) {
    this.id = id;
    this.tableNumber = tableNumber;
    this.capacity = capacity;
    this.available = available;
    this.restaurantId = restaurantId;
  }
}
